.github/workflows/push_bundles_catalog.yaml

```diff
@@ -1,5 +1,8 @@
 name: Build and release images
 
+env:
+  CONTAINER_TOOL: 'docker'
+
 on:
   push:
```

Comment:
Aside: I'm still a proponent of adding a `pull_request` here that does the build and skips the push, but at this point the push is pretty baked in and it's not crucial.

Reply:
I agree, and if `podman` gets updated in the ubuntu runner, this would be pretty good. But I think that since `docker` can't create a multi-arch manifest unless all of the individual images are pushed to a registry, skipping all of the pushes takes away a lot of the commands. Alternately, I guess we could have it running a local registry? Might be more trouble than it's worth though.